elastic/kibana
# [Alert details page] Use alert rule params instead of rule params in the custom threshold app section

Related to #181828

## Summary

This PR refactors the custom threshold app section to rely on the alert rule params instead of rule params.

### How to test
- Create a custom threshold rule and verify that the alert details page works as before
   - Also, check the log rate analysis component and ensure it works as before
